pkg/cmd/ssh/options.go

```diff
@@ -565,6 +565,43 @@ func (o *SSHOptions) Run(f util.Factory) error {
 	return err
 }
 
+func (o *SSHOptions) bastionIngressPolicies(shoot *gardencorev1beta1.Shoot) ([]operationsv1alpha1.BastionIngressPolicy, error) {
```

Comment:
This method only requires the providerType as input parameter. It would keep it simpler as it is an internal method we change change it if more info is needed.

Reply:
✅